tests/constraints/version/test_version_range.py

```diff
@@ -79,135 +79,68 @@ def v300b1() -> Version:
 
 
 @pytest.mark.parametrize(
-    "base,other",
+    ("constraint", "check_version", "allowed"),
     [
-        pytest.param(Version.parse("3.0.0-1"), Version.parse("3.0.0-1"), id="post"),
-        pytest.param(
-            Version.parse("3.0.0"), Version.parse("3.0.0+local.1"), id="local"
-        ),
+        # Inclusive ordering
```

Comment:
This is really an improvement. 👍  One minor thing: Can we add all cases where `check_version` is the version in the constraint, e.g. `<=3.0.0+local1` with `3.0.0+local1`, `>=3.0.0-1` with `3.0.0-1` and so on?